README.md

```diff
@@ -345,6 +345,10 @@ We've like to update this solution as version changes in `react-native-iap`.
 - Offical doc is [here](https://developer.android.com/google/play/billing/billing_library_overview).
 - I've developed this feature for other developers to contribute easily who are aware of these things. The doc says you can also get the `accessToken` via play console without any of your backend server. You can get this by following process.
   * Select your app > Services & APIs > "YOUR LICENSE KEY FOR THIS APPLICATION Base64-encoded RSA public key to include in your binary". [reference](https://stackoverflow.com/questions/27132443/how-to-find-my-google-play-services-android-base64-public-key).
+  
+#### How to make consumable product in android developer mode?
+- If you are facing `"You already own this item"` on developer(test) mode you might check this.
```

Comment:
I think it'd be better to make it in one sentence seen two are the same things. Like below.
If you are facing `"You already own this item"` on developer(test) mode you might check [related issue #126](https://github.com/dooboolab/react-native-iap/issues/126#issuecomment-439084872).